openmc/cell.py

```diff
@@ -79,13 +79,12 @@ class Cell(object):
     translation : Iterable of float
         If the cell is filled with a universe, this array specifies a vector
         that is used to translate (shift) the universe.
-    offsets : ndarray
-        Array of offsets used for distributed cell searches
-    distribcell_index : int
-        Index of this cell in distribcell arrays
-    distribcell_paths : list of str
-        The paths traversed through the CSG tree to reach each distribcell
-        instance
+    paths : list of str
+        The paths traversed through the CSG tree to reach each cell
+        instance. This property is initialized by calling the
+        :meth:`Geometry.determine_paths` method.
```

Comment:
Hmm, good point. I guess what I'll do is defer running that method to the user except when linking a summary file with a statepoint -- in that case, if a distribcell filter is found in a tally, `summary.geometry.determine_paths()` will get called automatically.

The other thought I had was to automatically determine paths if and when the user calls the getter for `Cell.paths`, but that would require backward links in the CSG tree which we don't currently have. In theory, we could implement that by having a `Cell._root` or `Cell._geometry` private attribute that gets set when a root universe is assigned to a geometry. But, that might be a little too magical (and it might seem weird to a user if the first time they call `Cell.paths` if takes 12 seconds or however long).